### DISCUSSION TITLE
Status Code: 502 Could not relay message upstream

### DISCUSSION THREAD

**Discussion by ale4ko69** [open-meteo/open-meteo#270] [Q&A]
Hello.
I have a question. When I switch through the city combobox between cities quickly, I get an error "Status Code: 502 Could not relay message upstream". After a short wait, it works again, but the next switching must wait again, otherwise, an error occurs.
![502](https://user-images.githubusercontent.com/14975911/228594657-cf19530e-04eb-4279-8e69-c1499fa30172.JPG)

**Comment by patrick-zippenfenig** [comment]
Around this time there were some abnormal high peaks in API calls per second and it blocked an upstream in nginx. I changed some limit in the nginx config and it should not block the upstream again.

Did you notice this issue at other times as well?

<img width="870" alt="Screenshot 2023-03-30 at 09 24 29" src="https://user-images.githubusercontent.com/11670456/228760689-73482895-dfb2-4de7-9b35-d5672b9ae5fc.png">

**Reply by ale4ko69** [reply]
Hi, this is unchanged. Look at the attached file...
![502_1](https://user-images.githubusercontent.com/14975911/229333538-efd9225e-84f3-436e-8c6a-9d6120c32547.JPG)

**Reply by patrick-zippenfenig** [reply]
I was monitoring all logs and barely see any errors. Do you use any proxies? If you notice it again, please include the exact time, so I can better trace it in the logs. Thanks
